addons/product/models/product_tag.py

```diff
@@ -15,6 +16,7 @@ def _get_default_variant_id(self):
         return self.env['product.product'].browse(self.env.context.get('product_variant_id'))
 
     name = fields.Char(string="Name", required=True, translate=True)
+    sequence = fields.Integer(string='Sequence')
```

Comment:
Shouldn't there be a default? Or it is ok to be the first one after creation?
Also, I know examples around it don't follow this remark but there is no need to define the string if it is the same as the name of the field.
```suggestion
    sequence = fields.Integer()
```